pkg/config/config.go

```diff
@@ -12,6 +12,7 @@ type Providers struct {
 	Qemu       []Qemu       `yaml:"qemu"`
 	Photon     []Photon     `yaml:"photon"`
 	Xen        []Xen        `yaml:"xen"`
+	Openstack  []Openstack  `yaml:"openstack"`
```

Comment:
I wonder if having `Openstack` instead of `OpenStack` is better? I see that Virtualbox does not capitalise `B`, however I am not quite sure whether it's because of some sort of naming convention or just mistyped.